services/121-service/src/transaction-job-processors/transaction-job-processors.service.ts

```diff
@@ -253,10 +252,22 @@ export class TransactionJobProcessorsService {
       }
     }
 
-    // 3. Start the transfer, save error transaction on failure and return early
-    let safaricomDoTransferResult: DoTransferReturnType;
+    // 3. Store 'waiting' transaction before starting transfer, because of callback.
+    const transaction = await this.createTransactionAndUpdateRegistration({
+      programId: input.programId,
```

Comment:
New convention: do not use "input." anymore. WIP as per Dom's suggestions. I can show/tell you now, or it comes in a next review.

Reply:
I have changed the `input` to `transactionJob`